Tests/SmokeDynamoDBTests/DynamoDBCompositePrimaryKeyTableUpdateItemConditionallyAtKeyTests.swift

```diff
@@ -706,20 +659,19 @@ class DynamoDBCompositePrimaryKeyTableUpdateItemConditionallyAtKeyTests: XCTestC
             XCTFail("Unexpected error thrown: \(error).")
         }
         
-        let secondRetrievedItem: TestTypeADatabaseItem? = try! table.getItem(forKey: key).wait()
+        let secondRetrievedItem: TestTypeADatabaseItem? = try await table.getItem(forKey: key)
         
         XCTAssertNil(secondRetrievedItem)
     }
     
-    func testUpdateItemConditionallyAtKeyWithUnknownItemWithAsyncItemProvider() {
-        let table = InMemoryDynamoDBCompositePrimaryKeyTable(eventLoop: eventLoop)
+    func testUpdateItemConditionallyAtKeyWithUnknownItemWithAsyncItemProvider() async throws {
```

Comment:
This test is now identical to `testUpdateItemConditionallyAtKeyWithUnknownItemWithItemProvider`.